pegasus/sites.v3/code.org/public/teacher-dashboard/signin_cards.haml

```diff
@@ -81,9 +81,12 @@ content-type: text/ng-template
   %img{"ng-src" => "/images/teacher-dashboard/GoogleSignInLink.png",
              alt: "Google Sign-in Link",
              title: "Google Sign-in Link",
-             :style => "border-color: #d8d8d8; border-width: 10px; border-style: solid; width: 250px; height:auto;"}
+             :style => "border-color: #d8d8d8; border-width: 10px; border-style: solid; width: 300px; height:auto;"}
   %h3= I18n.t('syncing_students')
   =I18n.t('syncing_students_instructions')
 
   %br
-  #react-sync-oauth-section{'ng-init' => '$emit("oauth-sync-react-rendered");'}
+  %img{"ng-src" => "/images/teacher-dashboard/syncGoogleClassroom.png",
```

Comment:
Should also remove [this hook](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/sites/code.org/pages/public/teacher-dashboard/index.js#L596-L609) that renders the button into the removed div - it's never executed now that this is removed.